### DISCUSSION TITLE
Where do we set the path to a local library?

### DISCUSSION THREAD

**Discussion by idavillc** [SVG-Edit/svgedit#474] [Q&A] [answered]
Where do we set the path for the local demo library so that it chooses the folder with all of our svg files in it to choose from?

**Comment by Lorrainbow** [comment, marked as answer]
I'm two years too late, but in case anyone else stumbles across this question, here's the answer from a django point of view. 

I took this code from index.html and put it in my django template page. You need to reference the css file as well but here's the editor.js edited and what they were looking for: setting the img_path to your path, in my case my django static path


```
<script type="module">        
import Editor from '{% static "svgedit/Editor.js" %}'
        
const svgEditor = new Editor(document.getElementById('container'))
svgEditor.init()
svgEditor.setConfig({
          dimensions: [320,240],
          canvas_expansion:5,
//ANSWER HERE 
          imgPath: '{% static "svgedit/images/" %}',
          allowInitialUserOverride: true,
          extensions: [],
          noDefaultExtensions: false,
          userExtensions: [/* { pathName: '/packages/react-test/dist/react-test.js' } */]
        })


```
